OpenEdXMobile/src/main/java/org/edx/mobile/util/images/ShareUtils.java

```diff
@@ -103,12 +105,15 @@ private String getSharingText(@NonNull ShareUtils.ShareType shareType) {
      * @param listener    Listener to take a callback when user select a particular app from
      *                    displayed popup.
      */
+    @RequiresApi(api = Build.VERSION_CODES.N)
```

Comment:
Please remove this and move the check internally because otherwise, we won't be able to call this function on lower devices.